packages/Webkul/Shop/src/Http/Controllers/Customer/AddressController.php

```diff
@@ -167,22 +151,18 @@ public function destroy($id)
     {
         $customer = auth()->guard('customer')->user();
 
-        $address = $this->customerAddressRepository->findOneWhere([
+        $this->customerAddressRepository->findOneWhere([
             'id'          => $id,
             'customer_id' => $customer->id,
         ]);
 
-        if (! $address) {
-            abort(404);
-        }
```

Comment:
We need this code as the upper value is nullable also.

Reply:
fixed